features/e2e_tests/README.md

```diff
@@ -0,0 +1,39 @@
+# E2E Tests
+
+## Test execution
+
+The way to execute the tests It's quite easy:
+
+- Input:
+```
+bash performance_test.sh
```

Comment:
Make the script executable and you won't need `bash performance_test.sh` but `./performance_test.sh` instead

Reply:
Both cases are covered, adding it to the doc